Microsoft.Toolkit.Uwp/Helpers/ScreenUnitHelper.cs

```diff
@@ -88,5 +90,17 @@ public static float Convert(ScreenUnit from, ScreenUnit to, float value)
                     throw new ArgumentOutOfRangeException(nameof(from));
             }
         }
+
+        private static float GetScale(XamlRoot xamlRoot)
```

Comment:
See my other comment about making some general UIElement extensions, wondering if we just could add this converter here to have context to a given UIElement even though it's more generalized it's most likely being called from that sort of context?